### DISCUSSION TITLE
guides/konteyner/dockerga-kirish

### DISCUSSION THREAD

**Discussion by giscus** [ismoilovdevml/devops-journey#38] [General]
# guides/konteyner/dockerga-kirish

Dockerga Kirish

https://devops-journey.uz/guides/konteyner/dockerga-kirish

**Comment by jamolweb** [comment]
zor !!!!!

**Comment by negmatov97** [comment]
Assalom aleykum! Ta'rif berishda salgina tushunarsizliklar bor ekan. Ammo, g'oya ajoyib! O'zbek tilida shunaqa kontentlarning bo'lishi xursand qilayapti! Mehnatingiz uchun rahmat!

**Comment by kaiunix17** [comment]
docker-compose haqida to'liqroq yozilsa yaxshi bo'lardi. Ammo shunisiga ham rahmat ancha foydali bo'ldi
